src/lib/models/reflections/declaration.ts

```diff
@@ -311,6 +311,7 @@ export class DeclarationReflection extends ContainerReflection {
                 this.implementedTypes,
             ),
             implementedBy: serializer.toObjectsOptional(this.implementedBy),
+            readme: Comment.serializeDisplayParts(serializer, this.readme),
```

Comment:
Can you update the deserializer to handle the readme property unconditionally too?

Reply:
Done, I believe this to be sufficient -- the changes are working with my monorepo as expected.

Thanks also for the prompt feedback, as usual! 😄